packages/examples/examples/wasm/package.json

```diff
@@ -15,7 +15,7 @@
   ],
   "scripts": {
     "prebuild": "yarn run build:wasm",
-    "build": "yarn run prebuild && mm-snap build",
+    "build": "mm-snap build",
```

Comment:
Hah, I didn't realize `pre` and `post` scripts worked for _all_ scripts, not just specific instances like `install` / `postinstall`. Nice.

Ref: https://docs.npmjs.com/cli/v6/using-npm/scripts